conda/recipes/libcuml/conda_build_config.yaml

```diff
@@ -4,6 +4,9 @@ cmake_version:
 nccl_version:
   - ">=2.9.9"
 
+ucx_version:
+  - "1.12.1"
```

Comment:
I think this should be `>=1.12.1`

Reply:
Same comment as here ( https://github.com/rapidsai/raft/pull/638#discussion_r864537580 ).